e2e/interchaintestv8/types/rust_fixtures.go

```diff
@@ -0,0 +1,61 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	clienttypes "github.com/cosmos/ibc-go/v9/modules/core/02-client/types"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
+	ethereumlightclient "github.com/srdtrk/solidity-ibc-eureka/e2e/v8/types/ethereumlightclient"
+)
+
+type CommitmentProofFixture struct {
+	Path           []byte                             `json:"path"`
+	StorageProof   ethereumlightclient.StorageProof   `json:"storage_proof"`
+	ProofHeight    clienttypes.Height                 `json:"proof_height"`
+	ClientState    ethereumlightclient.ClientState    `json:"client_state"`
+	ConsensusState ethereumlightclient.ConsensusState `json:"consensus_state"`
+}
+
+type RustFixtureGenerator struct {
+	shouldGenerateFixture bool
+	prefix                string
+
+	// fixtureCount is used to create a clear order of fixtures
+	fixtureCount uint
```

Comment:
Probably. I can look into that. Maybe just keeping the different fixture writes in memory and writing them all out to a single file per test at the end.

Reply:
Actually, I'll do this in https://github.com/cosmos/solidity-ibc-eureka/issues/143 since the whole fixture setup will be part of that. Added it to the list